### PR TITLE
[WIP] begin updating synthesis vignette

### PR DESCRIPTION
**Pull request by dlebauer**
resolving #121

Need to 

* clarify explanation of the logistic growth model
* make sure that we are consistently using season 6 / 2018 canopy cover and weather data
* might need to update the images that we are using

### CODE REVIEW

File: vignettes/04-synthesis-data.Rmd [hunk 1]
```diff
@@ -9,8 +9,8 @@ The second analysis compares greenness from image data with canopy cover.
 ## Get and join data
 
 Here we combine two dataframes. 
-The first contains all the canopy height values for 2017, which was created in the traits vignette. 
-The second is the cumulative growing degree days for all of 2017, which were calculated from the daily minimum and maximum temperatures in the weather vignette. 
+The first contains all the canopy cover values for 2018, which was created in the traits vignette.
```
Comment (KristinaRiemer) [inline vignettes/04-synthesis-data.Rmd]: @dlebauer weather and canopy cover data for 2018 don't overlap. Weather data is only currently available for January - April 2018 while canopy cover data is for May - July 2018.

Comment (KristinaRiemer) [reply vignettes/04-synthesis-data.Rmd]: And canopy cover data is only in May for 2017, though there is full weather data for that year.

Comment (dlebauer) [reply vignettes/04-synthesis-data.Rmd]: Can you ask @tcnichol to check on the 2018 weather data? It should be available in geostreams.

**Comment by KristinaRiemer** [comment]
@dlebauer I can't figure out why the inflection points aren't correct, see plot generated from line 160.

File: vignettes/04-synthesis-data.Rmd (hunk 1 above)
Comment (KristinaRiemer) [reply vignettes/04-synthesis-data.Rmd]: 2018 weather data in now available.

**Comment by KristinaRiemer** [comment]
@kimberlyh66 would you be able to make sure this vignette knits properly and looks okay?

**Comment by KristinaRiemer** [comment]
Also, the image files take a while to download, so I don't know if it would be possible to use a smaller subset of these or find a way to speed up that download? That would be an issue to address later on.

**Comment by KristinaRiemer** [comment]
@kimberlyh66 done!

**Comment by kimberlyh66** [comment]
@KristinaRiemer sure I will review and merge

**Pull request merged**